src/pages/login.tsx

```diff
@@ -12,13 +12,19 @@ const Login = () => {
       </Link>
       <button
         className="px-4 py-2 mb-4 text-white bg-black rounded hover:bg-gray-800"
-        onClick={() => void signIn('github')}
+        onClick={() => {
+          {/* eslint-disable-next-line @typescript-eslint/no-floating-promises */}
```

Comment:
noob question: what are the curly braces around the comment for?

Reply:
ok, just checking because your eslint-disable comment in auth.ts does not have the braces